vent/api/actions.py

```diff
@@ -683,26 +683,42 @@ def get_configure(self,
                       groups=None,
                       enabled="yes",
                       branch="master",
-                      version="HEAD"):
+                      version="HEAD",
+                      main_cfg=False):
         """
         Get the vent.template settings for a given tool by looking at the
         plugin_manifest
         """
         self.logger.info("Starting: get_configure")
         constraints = locals()
         status = (True, None)
-        # all possible vent.template sections
-        options = ['info', 'service', 'settings', 'docker', 'gpu']
-        tools = self.p_helper.constraint_options(constraints, options)[0]
-        if tools:
-            # should only be one tool
-            tool = tools.keys()[0]
-            # load all vent.template options into dict
-            template_dict = {}
-            for section in tools[tool]:
-                template_dict[section] = json.loads(tools[tool][section])
-            # display all those options as they would in vent.template
-            return_str = ""
+        template_dict = {}
+        return_str = ""
+        if main_cfg:
+            vent_cfg_path = os.path.join(os.path.expanduser('~'), '.vent',
```

Comment:
@bpagon13 looks like this one got missed in your update, I think it's the last one.